lib/src/main/java/com/auth0/jwt/impl/PayloadDeserializer.java

```diff
@@ -65,9 +65,12 @@ public Payload deserialize(JsonParser p, DeserializationContext ctxt) throws IOE
 
     Date getDateFromSeconds(Map<String, JsonNode> tree, String claimName) {
         JsonNode node = tree.get(claimName);
-        if (node == null || node.isNull() || !node.canConvertToLong()) {
+        if (node == null || node.isNull()) {
```

Comment:
So non Java person here, what is the difference here in checking for null?

Reply:
`node` is the instance of the `JsonNode` we're checking. The check is asserting that the instance of this object is `null` OR that the parsed JSON value is null. This second check wouldn't be possible if the instance is `null` as that would throw a NPE when calling some method on it.